modules/core/test/v2/unit/coins/sol.ts

```diff
@@ -165,6 +174,55 @@ describe('SOL:', function () {
       const validTransaction = await basecoin.verifyTransaction({ txParams, txPrebuild, memo, durableNonce });
       validTransaction.should.equal(true);
     });
+    it('should verify activate staking transaction', async function () {
```

Comment:
there is no test for deactivate builder. 
Also there are not failure cases. Please try to add failure cases.

Reply:
This is covered from here "line 136", to here "line 171 "